src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -380,6 +392,15 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 					id: index,
 					name: tag,
 				}));
+				outerThis.tagOptions = outerThis.existingTags.map(
+					(tag) =>
+						({
+							//@ts-ignore
```

Comment:
Why is this ignore comment necessary?

Reply:
removed it and used a better type for existingTags